alibi/explainers/anchor_tabular.py

```diff
@@ -937,13 +964,13 @@ def add_names_to_exp(self, explanation: dict) -> None:
                 if geq > float('-inf'):
                     if geq == len(self.feature_values[feat_id]) - 1:
                         geq = geq - 1
-                    name = self.feature_values[feat_id][geq + 1]
+                    name = self.feature_values[feat_id][int(geq) + 1]
                     if '<' in name:
                         geq_val = name.split()[0]
                     elif '>' in name:
                         geq_val = name.split()[-1]
                 if leq < float('inf'):
-                    name = self.feature_values[feat_id][leq]
+                    name = self.feature_values[feat_id][int(leq)]
```

Comment:
Under what circumstances do these need to be cast to `int` explicitly? What change in this PR made this casting necessary?

Reply:
This is due to `mypy` complaining as the `category_map` is now fully typed with integer keys.